addons/common/functions/fnc_setupLocalUnitsHandler.sqf

```diff
@@ -0,0 +1,76 @@
+/*
+ * Author: dedmen  
+ * Adds the local unit event handlers. Access with array `ace_common_localUnits`.
+ *
+ * Arguments:
+ * Nothing
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [] call ace_common_fnc_setupLocalUnitsHandler
+ *
+ * Public: No
+ */
+// #define DEBUG_MODE_FULL
+#include "script_component.hpp"
+
+GVAR(localUnits) = [];
+
+//Eventhandlers to maintain array of localUnits
+["CAManBase", "init", {
+    params ["_unit"];
+    TRACE_2("unit init",_unit,local _unit);
+
+    if (local _unit) then {
+        if (!alive _unit) exitWith {};
+        GVAR(localUnits) pushBack _unit;
+    };
+}] call CBA_fnc_addClassEventHandler;
+
+["CAManBase", "respawn", {
```

Comment:
Shouldn't "init" also fire for respawning units?
But I guess they might be !alive at init? 

Respawning in MP should be tested.

Reply:
Init does not fire for respawned objects.